spryker/usr/local/share/spryker/spryker_functions.sh

```diff
@@ -156,3 +156,12 @@ do_spryker_setup_search() {
 do_spryker_run_tests() {
   as_code_owner "vendor/bin/codecept run --skip Acceptance"
 }
+
+do_spryker_console() {
```

Comment:
As `set +x` is in use on line 161, can we change this to be a subshell execution instead, e.g.
```do_spryker_console() (
  # ...
)``` rather than the `{ }`

Reply:
I don't get that

Reply:
Avoids the need to reset the turning off of tracing with `set -x` later on, by running everything for the function in a subshell ( see https://github.com/continuouspipe/dockerfiles/pull/365 )